# Exercise No.30

# Why is there an error here?
# def foo(a=2, b):
# 	return a + b



# Solution
# Because default arguments must be listed at the end of the parameters list. If you list them at the beginning python
# will get confused because it is not sure if you are changing the default argument's value or passing the second
# argument.



